core/pom.xml

```diff
@@ -72,13 +72,19 @@
 		<dependency>
 			<groupId>com.azure</groupId>
 			<artifactId>azure-security-keyvault-keys</artifactId>
-			<version>4.3.4</version>
+			<version>4.4.6</version>
 			<optional>true</optional>
+			<exclusions>
+				<exclusion>
```

Comment:
Why did you add this exclusion?

Reply:
There's no upstream fix for vulnerabilities in that library, but the unit tests all passed without it.

Reply:
It's also compiling fine if you don't exclude this dependency. I get that it's not needed, but there seems to be a lot of transient dependencies for `azure-security-keyvault-keys` that are not used by this project. Seems arbitrary to exclude this single one.

Reply:
> > > I'm getting a build error when trying to compile the core library on this branch. I'm not getting this error on master.
> > > ```
> > > [INFO] ------------------------------------------------------------------------
> > > [ERROR] Failed to execute goal org.owasp:dependency-check-maven:7.1.1:check (default) on project java-saml-core: One or more exceptions occurred during dependency-check analysis: One or more exceptions occurred during analysis:
> > > [ERROR] 	InitializationException: Warn initializing the suppression analyzer: Failed to load .nvd-suppressions.xml, caused by Unable to read suppression file. 
> > > [ERROR] 		caused by SuppressionParseException: Failed to load .nvd-suppressions.xml, caused by Unable to read suppression file. 
> > > [ERROR] 	InitializationException: Warn initializing the suppression analyzer: Failed to load .nvd-suppressions.xml, caused by Unable to read suppression file. 
> > > [ERROR] 		caused by SuppressionParseException: Failed to load .nvd-suppressions.xml, caused by Unable to read suppression file.
> > > [ERROR] -> [Help 1]
> > > ```
> > 
> > 
> > Try doing a `mvn clean` if you haven't already. I had the same error until I did that.
> 
> I have tried that, but it's not helping. The issue seems to be related the updated version of org.owasp:dependency-check-maven, not looking for `.nvd-suppressions.xml` in the right place. You may want to specify the path to this file more precise.
> 
> To be clear, I'm talking about comping the core module by itself, not the whole multi module project.

Pushed a change that pulls the suppression file directly from github instead of path shenanigans.

Reply:
I removed the exclusion.